src/Components/AddProject/AddProject.tsx

```diff
@@ -51,7 +54,7 @@ axios.post(`${baseUrl}/Project`,data,{headers:reqHeaders}).then((response)=>{
     
     <label className='d-block' >Description</label>
   
-   <textarea className='form-control  border-2 rounded-4' cols="20" rows="4"   {...register("description",{required:true})} >Description
+   <textarea className='form-control  border-2 rounded-4' cols="20" rows="4" placeholder='Description'   {...register("description",{required:true})} >
```

Comment:
Consider validating any input field for better user experience and to add an additional layer of security